scripts/check_version

```diff
@@ -36,13 +56,36 @@ if [[ x${STRIPPED_PACKAGE_VERSION}x != x${IOS_MARKETING_VERSION_2}x ]]; then
 	EXITCODE=-1
 fi
 
-# This will check the format of IOS' CURRENT_PROJECT_VERSION
-if [[ ! "$IOS_PROJECT_VERSION_1" =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
-    echo "Invalid format (expected X.X.X) for first CURRENT_PROJECT_VERSION (${IOS_PROJECT_VERSION_1}) on ios/HathorMobile.xcodeproj/project.pbxproj"
+# Check the format of iOS' CURRENT_PROJECT_VERSION
+# If it's a release version (without hyphen in package version), it should be 1.0.0
+# If it's a release candidate version (with hyphen in package version), it should be 0.rc.0, where rc is taken from package version
+if [ -z "$STRIPPED_PACKAGE_CANDIDATE" ]; then
+  if [[ x${IOS_PROJECT_VERSION_1}x != x1.0.0x ]]; then
```

Comment:
[non-blocking] Why are you wrapping with "x"?

Reply:
That was to keep with the pattern for other string comparisons already on this script. All of them compare with a `x` wrapping.